src/app/api/peanut/submit-cashout-link/route.ts

```diff
@@ -24,13 +25,14 @@ export async function POST(request: NextRequest) {
                 'api-key': apiKey,
             },
             body: JSON.stringify({
-                bridgeCustomerId: bridgeCustomerId,
-                liquidationAddressId: liquidationAddressId,
-                cashoutTransactionHash: cashoutTransactionHash,
-                externalAccountId: externalAccountId,
-                chainId: chainId,
-                tokenName: tokenName,
-                pubKey: pubKey,
```

Comment:
todo: add trackParam to call (with the same value as the promoToken for now)

Reply:
> todo: add trackParam to call (with the same value as the promoToken for now)

@jjramirezn  need to pass both `promoCode` and `trackParam` with same values you mean to say?